src/riak_kv_ts_api.erl

```diff
@@ -46,15 +46,16 @@
 -include("riak_kv_ts.hrl").
 
 %% external API calls enumerated
--type query_api_call() :: query_create_table | query_select | query_describe | query_insert.
+-type query_api_call() :: query_create_table | query_select | query_describe | query_insert | query_show_tables.
 -type api_call() :: get | put | delete | list_keys | coverage | query_api_call().
 -export_type([query_api_call/0, api_call/0]).
 
 -spec api_call_from_sql_type(riak_kv_qry:query_type()) -> query_api_call().
-api_call_from_sql_type(ddl)      -> query_create_table;
-api_call_from_sql_type(select)   -> query_select;
-api_call_from_sql_type(describe) -> query_describe;
-api_call_from_sql_type(insert)   -> query_insert.
+api_call_from_sql_type(ddl)         -> query_create_table;
```

Comment:
This function should be like the snippet below to avoid big diffs like this.

```erlang
api_call_from_sql_type(ddl) ->
    query_create_table.
```